talkgenerator/utils.py

```diff
@@ -120,9 +125,9 @@ def str2bool(v):
 
 def get_argument_parser():
     parser = argparse.ArgumentParser(description='Quickly build a slide deck.')
-    parser.add_argument('--topic', default='cat', type=str,
+    parser.add_argument('--topic', default='', type=str,
                         help="Topic of presentation.")
-    parser.add_argument('--num_slides', default=10, type=int,
+    parser.add_argument('--num_slides', '--slides', default=10, type=int,
```

Comment:
@twinters ... is there a reason we use multiple keyword arguments here?

Reply:
I find "slides" to be a more natural name for this parameter, but don't want to break existing installations.

Reply:
Cool!